### PR TITLE
Document quarkus.uuid usage in the kafka reference guide

### PR DESCRIPTION
**Pull request by cescoffier**
Document the usage of ${quarkus.uuid} to get a unique Kafka consumer group every time

### CODE REVIEW

File: docs/src/main/asciidoc/kafka.adoc [hunk 1]
```diff
@@ -539,6 +539,23 @@ Or have the listener’s name be the same as the group id:
 
 Setting the consumer re-balance listener’s name takes precedence over using the group id.
 
+==== Using unique consumer groups
+
+To always process the records from a topic from the first one, you need a unique consumer group.
```
Comment (ozangunalp) [inline docs/src/main/asciidoc/kafka.adoc:544]: I suggest :
```suggestion
If you want to process records from a topic from its beginning, you need to set `auto.offset.reset = earliest` **and** create a new consumer group, with an unused `group.id`. 
Quarkus generates a UUID that changes between two executions (including in dev mode). 
So you are sure no other consumer group uses it and you receive a new unique group id every time your application starts.
```

**Review by ozangunalp — CHANGES_REQUESTED** [review]

**Review by gsmet — CHANGES_REQUESTED** [review]
The text is a bit weird, typically you have `Quarkus generates a UUID that changes between two executions (including in dev mode).` twice.

And having `So, you are sure no other consumer uses it` just after `So you are sure no other consumer group uses it` also reads funny.

File: docs/src/main/asciidoc/kafka.adoc [hunk 2]
```diff
@@ -539,6 +539,25 @@ Or have the listener’s name be the same as the group id:
 
 Setting the consumer re-balance listener’s name takes precedence over using the group id.
 
+==== Using unique consumer groups
+
+If you want to process records from a topic from its beginning, you need to set `auto.offset.reset = earliest` **and** create a new consumer group, with an unused `group.id`. 
+Quarkus generates a UUID that changes between two executions (including in dev mode). 
+So you are sure no other consumer group uses it and you receive a new unique group id every time your application starts.
+So, you are sure no other consumer uses it (so your consumer is the only one in the group).
```
Comment (ozangunalp) [inline docs/src/main/asciidoc/kafka.adoc:547]:
```suggestion
```

File: docs/src/main/asciidoc/kafka.adoc [hunk 3]
```diff
@@ -539,6 +539,25 @@ Or have the listener’s name be the same as the group id:
...
+You also need to receive a new value every time you start your application.
```
Comment (ozangunalp) [inline docs/src/main/asciidoc/kafka.adoc:548]:
```suggestion
```

File: docs/src/main/asciidoc/kafka.adoc [hunk 4]
```diff
@@ -539,6 +539,25 @@ Or have the listener’s name be the same as the group id:
...
+
+Quarkus generates a UUID that changes between two executions (including in dev mode).
```
Comment (ozangunalp) [inline docs/src/main/asciidoc/kafka.adoc:550]:
```suggestion
```

**Review by ozangunalp — CHANGES_REQUESTED** [review]
@gsmet true :) 

I think @cescoffier applied my change suggestion as is. I was thinking of removing the following lines.

**Comment by cescoffier** [comment]
I've reformatted the paragraph to avoid duplications.

**Review by ozangunalp — APPROVED** [review]

**Review by gsmet — APPROVED** [review]

**Pull request merged**